P202.Training.Data/EntityBase.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
```

Comment:
Is this class being used? I think the one being used is the one inside the "Entities" folder

Reply:
i exclude from Project this file, i delete for the repository